impl/src/main/java/org/ehcache/config/builders/CacheConfigurationBuilder.java

```diff
@@ -501,7 +447,7 @@ public boolean hasConfiguredExpiry() {
    * @return a new builder with the added configuration
    */
   public CacheConfigurationBuilder<K, V> withDispatcherConcurrency(int dispatcherConcurrency) {
```

Comment:
Should the other `with` methods have corresponding `without` methods?

Reply:
Yes... although many of them should be `withDefault<X>` methods. Added.